browser/main/lib/ConfigManager.js

```diff
@@ -58,11 +62,21 @@ function get () {
   let config = window.localStorage.getItem('config')
 
   try {
+    const boostnotercConfig = RcParser.parse()
+
     config = Object.assign({}, DEFAULT_CONFIG, JSON.parse(config))
+
+    config = Object.assign({}, DEFAULT_CONFIG, boostnotercConfig)
+    config.hotkey = Object.assign({}, DEFAULT_CONFIG.hotkey, boostnotercConfig.hotkey)
+    config.ui = Object.assign({}, DEFAULT_CONFIG.ui, boostnotercConfig.ui)
+    config.editor = Object.assign({}, DEFAULT_CONFIG.editor, boostnotercConfig.editor)
+    config.preview = Object.assign({}, DEFAULT_CONFIG.preview, boostnotercConfig.preview)
+
     config.hotkey = Object.assign({}, DEFAULT_CONFIG.hotkey, config.hotkey)
```

Comment:
Can't you write like below?

```javascript
config.hotkey = Object.assign({}, DEFAULT_CONFIG.hotkey, boostnotercConfig.hotkey, config.hotkey);
```

And I think this code doesn't implement with DRY principle. So maybe, create function to bind config and using it best!(You don't need to do that at this PR. Let's challenge next time :) )